scripts/fetchdep.sh

```diff
@@ -25,30 +25,37 @@ clone() {
 # First we check if GITHUB_HEAD_REF is defined,
 # Then we check if BUILDKITE_BRANCH is defined,
 # if it isn't we can assume this is a Netlify build
-if [ -n ${GITHUB_HEAD_REF+x} ]; then
-	head=$GITHUB_HEAD_REF
-elif [ -n ${BUILDKITE_BRANCH+x} ]; then
-	head=$BUILDKITE_BRANCH
+if [ -z ${BUILDKITE_BRANCH+x} ]; then
```

Comment:
Hmm, at first glance, it seems a lot more confusing to follow when structuring each check as what is _not_ present... Why is that needed...? I would have though each env var only appeared on each service, i.e. `GITHUB_HEAD_REF` appears only on GHA and `BUILDKITE_BRANCH` appears only on Buildkite...?